lib/src/util.dart

```diff
@@ -66,7 +66,22 @@ String longName(Info info, {bool useLibraryUri: false}) {
     // assert(!first || segment is LibraryInfo);
     // (today might not be true for for closure classes).
     if (segment is LibraryInfo) {
-      sb.write(useLibraryUri ? segment.uri : segment.name);
+      if (useLibraryUri && forId && segment.uri.isScheme('file')) {
+        var currentBase = Uri.base.toString();
```

Comment:
Uri.base.path

Reply:
We assume `Uri.base` is ALWAYS a `file:` URI then, right?